backend/tracim_backend/views/frontend.py

```diff
@@ -66,7 +66,11 @@ def bind(self, configurator: Configurator) -> None:
         # index.html for /index.html and /
         configurator.add_route('root', '/', request_method='GET')
         configurator.add_view(self.index, route_name='root')
-        configurator.add_route('ui', urljoin('/', FRONTEND_UI_SUBPATH, '.*'), request_method='GET')
+        configurator.add_route(
```

Comment:
- rename `path` slug to `ui_subpath` because *path* is too generic and it will be difficult for a developer to understand its real goal 
- put

Reply:
Done